src/cuttlefish.erl

```diff
@@ -123,7 +123,7 @@ invalid(Reason) ->
 %% being logged.
 -spec warn(iodata()) -> ok.
 warn(Str) ->
-    _ = lager:warning(Str, []).
+    _ = logger:warning(Str, []).
```

Comment:
The `?LOG_*()` macros should be used instead of a function call. They will take care of adding metadata automatically.

Also, we should specify the domain for every log message. So something like:
```erlang
?LOG_WARNING(Str, [], #{domain => [cuttlefish]}).
```

Reply:
OK, let's use those